site/content/en/docs/manual/basics/creating_an_annotation_task.md

```diff
@@ -1,130 +1,178 @@
+Added attributes will be available from the Objects menu:
 
-- The `Raw` is a way of working with labels for an advanced user.
-  Raw presents label data in _json_ format with an option of editing and copying labels as a text.
-  The `Done` button applies the changes and the `Reset` button cancels the changes.
-  ![](/images/image126.jpg)
+![Attributes](/images/attributes.jpg)
+
+To add an attribute, do the following:
+
+1. In the **Name** field add the attribute’s name.
+2. From the drop-down, select way to display the attribute:
+   - `Radio` to choose one of several options.
+   - `Checkbox` to choose several options.
+   - `Text` to add text attribute.
+   - `Number` to add number attribute.
+3. Set values for the attribute. To separate values use **Enter**.
+   To delete the added value, use **Backspace** or near the name of the value, click **x**.
+   > ** Note:** If the specified way of displaying the attribute is `Text` or `Number`,
+   > the entered value will be displayed as text by default (you can specify the text format).
+4. Select **Mutable** if an attribute would be changed from frame to frame.
+
+To delete an attribute, click **Delete attribute**.
 
 ### Select files
 
-Press tab `My computer` to choose some files for annotation from your PC.
-If you select tab `Connected file share` you can choose files for annotation from your network.
-If you select ` Remote source` , you'll see a field where you can enter a list of URLs (one URL per line).
-If you upload a video or dataset with images and select `Use cache` option, you can attach a `manifest.jsonl` file.
-You can find how to prepare it [here](/docs/manual/advanced/dataset_manifest/).
-If you select the `Cloud Storage` tab, you can select a cloud storage (for this type the cloud storage name),
-after that choose the manifest file and select the required files.
-For more information on how to attach cloud storage, see [attach cloud storage](/docs/manual/basics/attach-cloud-storage/)
+There are several ways to upload files for annotation:
+
+- To upload the file from your PC, go to the **My computer** tab.
+- To upload files from a connected file share or network, go to the **Connected file share** tab.
+- To upload files from remote sources, go to the **Remote source** tab. Enter a list of URLs (one
+  per line). If you upload a video or dataset with images and select the **Use cache** option, you
+  can attach a `manifest.json` file.
+  <br> On how to prepare the `manifest.json` file, see [Dataset manifest.](/docs/manual/advanced/dataset_manifest/)
+- To upload files from cloud storage, go to the **Cloud Storage** tab. Type the cloud storage name,
+  and after that choose the manifest file and select the required files.
+  <br> For more information, see [Attach cloud storage](/docs/manual/basics/attach-cloud-storage/)
+
+### RAW
+
+The **Raw** is a way of working with labels for an advanced user.
```

Comment:
Let's add here: 

**Important note:**

**Be careful** with changing raw specification of an existing task/project. Removing any "id" properties will lead to losing existing annotations. **This property will be removed automatically from any text you insert to this field.**

Reply:
added